src/DateTimePicker.jsx

```diff
@@ -98,15 +113,20 @@ class DateTimePicker extends React.Component {
 		return (datetime.toTimeString().split(' ')[0]).split(':', 2).join(':'); // TODO localize toLocaleTimeString ?
 	}
 
+	parseNewTimeAsDate(timeStr) {
```

Comment:
Actually, looking more closely at the docs, the date formatting advantages are mostly in the FormattedDate component. Using the actual API from react-intl requires injecting it into the component, which created a host of testing problems when I first attempted it. This is probably why I ended up looking into using moment instead.

The `timeStr` returned does not return time zone. It's the time value on the time input in 24hr time. Adding docs now so that's clearer.

I'm going to look into doing this without moment again. Would prefer not to inject react-intl. Will see if I can do it without that, too.